.gitignore

```diff
@@ -8,6 +8,7 @@
 /reports
 /config/config.json
 /config/config.local.json
+latest.sql
```

Comment:
I guess you will need to remove `latest.sql` file from the commit. This file is unlrelated to the issue. You can keep this file outside the bookbrainz-site directory so that git doesn't tracks it.